repository-data/webfiles/src/main/resources/site/scss/intranet/components/_intra-comments-blob.scss

```diff
@@ -0,0 +1,28 @@
+// Intra comments blob
+.intra-comments-blob {
+    @include clearfix;
+    @include bold-font(14);
+
+    display: inline-block;
+    color: $col-intranet-mid-grey;
+
+    svg {
```

Comment:
Have you seen the SVG Magic servlet (src/main/java/uk/nhs/digital/svg/colour/SvgColourMagicServlet.java)?

Example usage:
```html
<img src="${leadImage?replace("/binaries", "/svg-magic/binaries")}?colour=005eb8" ... />
```

Reply:
Yeah, that's great but it generates an extra HTTP request - same as above comment about the inline SVGs.